### PR TITLE
adguardhome: add option config

### PR DESCRIPTION
**Pull request by peterwillcn**
add option config for adguardhome.yaml

```option config /etc/adguardhome.yaml```

### CODE REVIEW

File: net/adguardhome/files/adguardhome.init [hunk 1]
```diff
@@ -22,11 +22,12 @@ start_service() {
 
   config_load adguardhome
   config_get WORK_DIR config workdir
+  config_get CONFIG_FILE config config
```
Comment (stangri) [inline net/adguardhome/files/adguardhome.init:25]: Maybe better to use:
```
config_get CONFIG_FILE config config '/etc/adguardhome.yaml'
```
to ensure default value is set if that option is not present (like when upgrading existing package with existing config)?

Comment (peterwillcn) [reply net/adguardhome/files/adguardhome.init]: fixed

**Comment by 1715173329** [comment]
should raise `PKG_RELEASE`

**Comment by peterwillcn** [comment]
> should raise `PKG_RELEASE`

Fixed

**Review by stangri — APPROVED** [review]

**Comment by peterwillcn** [comment]
ping @1715173329

File: net/adguardhome/files/adguardhome.init [hunk 2]
```diff
@@ -22,11 +22,12 @@ start_service() {
 
   config_load adguardhome
   config_get WORK_DIR config workdir
+  config_get CONFIG_FILE config config '/etc/adguardhome.yaml'
```
Comment (1715173329) [inline net/adguardhome/files/adguardhome.init:25]: use double quote here to keep code consistent

**Review by 1715173329 — APPROVED** [review]

**Pull request merged**